fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -192,21 +193,44 @@ public Response options() {
      * Retrieve the node profile
      *
      * @param rangeValue the range value
-     * @return triples for the specified node
+     * @return the binary or the triples for the specified node
      * @throws IOException if IO exception occurred
      */
     @GET
-    @Produces({TURTLE + ";qs=10", JSON_LD + ";qs=8",
-            N3, N3_ALT2, RDF_XML, NTRIPLES, APPLICATION_XML, TEXT_PLAIN, TURTLE_X,
-            TEXT_HTML, APPLICATION_XHTML_XML, "*/*"})
```

Comment:
I'm not sure where to go with this now, other then maybe going back a commit or two - to what @acoburn suggested before ([comment](https://jira.duraspace.org/browse/FCREPO-1840?focusedCommentId=47464&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-47464)) - my original commit on this pull request.  It's clear that there's a bigger design issue here that shouldn't probably be solved via this bug.  One thought is to change the method name 'describe' to something like 'getResource' so it's clear (to developers) that it's fetching NonRdfResources as well as RdfResources.  I did put it in the method's description, so maybe that's enough. 

I don't know how to leverage more JaxRS here.  I tried to see if I could break up the method into two (sub) methods, each with their own @Produces [here](https://gist.github.com/bseeger/d2c48ba1ac0a2e37ab1d). 

I predicted this wouldn't work, and it hasn't (at least not yet, I'm still testing it out).  I couldn't find anything online specifically saying this wouldn't work, though it does seem unlikely.